packages/controller/src/lib/letsencrypt.js

```diff
@@ -13,6 +13,10 @@ const fs = require('fs-extra');
  * @returns {Promise<unknown>}
  */
 async function createServerAsync(app, settings, certificates, leSettings, log, adapter) {
+    log.warn(
+        `Information for Developer: Using "Let's encrypt" is deprecated and will be removed in the next js-controller version, use @iobroker/webserver instead`
```

Comment:
Using the direct "Let's encrypt" module import is deprecated ... !?

might else sound like we kill the whole feature, same below